internal/test/wsu/wsu_test.go

```diff
@@ -87,6 +87,10 @@ func TestWSU(t *testing.T) {
 
 	// Run cluster wide tests
 	t.Run("Pending CSRs were approved", testNoPendingCSRs)
+
+	t.Run("Tests across Windows nodes", func(t *testing.T) {
+		testAcrossWindowsNodes(t)
```

Comment:
You dont need to wrap this function, you can just do:
```t.Run("Tests across Windows nodes",testAcrossWindowsNodes)```

Reply:
Fixed it